pkg/adaptor/adaptor.go

```diff
@@ -108,6 +108,18 @@ func (c *Config) splitNamespace() (string, string, error) {
 	return fields[0], fields[1], nil
 }
 
+// compileNamespace split's on the first '.' and then compiles the second portion to use as the msg filter
+func (c *Config) compileNamespace() (string, *regexp.Regexp, error) {
```

Comment:
would it make things cleaner to have a 
`type Namespace regexp.Regexp`
and then have methods `func NewNamespace(s string)`, and `func (ns *Namespace) Match(s) bool `